python/lsst/meas/algorithms/pcaPsfDeterminer.py

```diff
@@ -369,7 +393,8 @@ def determinePsf(self, exposure, psfCandidateList, metadata=None, flagKey=None):
                         badCandidates.append(cand)
 
             badCandidates.sort(key=lambda x: x.getChi2(), reverse=True)
-            numBad = int(len(badCandidates) * (iter + 1) / self.config.nIterForPsf + 0.5)
+            numBad = numCandidatesToReject(len(badCandidates), iter,
```

Comment:
It's icky, but I was reluctant to change it since it's not really related to this ticket. I don't suppose it'll do any harm, so maybe I should just go ahead.